.travis.yml

```diff
@@ -1,34 +0,0 @@
-sudo: false
-language: java
-after_success: ./.publish_site.sh
-env:
-  global:
-    # this contains the private ssh key for publishing documentation
-    # awk 'BEGIN{i=0} {i=i+1; system(sprintf("travis encrypt DEPLOY_KEY_%02d=\\\"%s\\\"\n", i, $0))}'
```

Comment:
@wvengen, to be safe, in case this ssh key is still used somewhere would be good to replace or delete it.

Reply:
Thank you!